index.bs

```diff
@@ -2554,12 +2554,45 @@ as it implies that it's possible to dispatch an event asynchronously.
 All events are dispatched synchronously.
 What is more often implied by "asynchronous event" is to defer firing an event.
 
-<h3 id="state-and-subclassing">Use plain {{Event}}s for state</h3>
+<h3 id="state-and-subclassing">Put state on {{Event/target}} objects rather than {{Event}}s</h3>
 
-Where possible, use a plain {{Event}} with a specified `type`,
-and capture any state information in the {{Event/target}} object.
+Where possible, rather than put state on {{Event}} objects, put state on the {{Event/target}}
+and use events to signal updates to that state.
 
-It's usually not necessary to create new subclasses of {{Event}}.
+<div class="note">
```

Comment:
Does this need to be a note?

Reply:
I did it to separate the rule from the explanation of the rule, since the explanation is non-normative. But this spec doesn't have such a strict divide between normative and non-normative. But I don't mind either way, happy to go with whatever you think.

Reply:
I'm not sure if I understand this paragraph...

Or if it's completely accurate. There are a few events that are fired asynchronously, e.g. `slotchange`. The state on the event target might have been updated when the event listener is triggered.